packages/victory-core/src/victory-label/victory-label.js

```diff
@@ -1,14 +1,16 @@
 import React from "react";
 import PropTypes from "prop-types";
 import VictoryPortal from "../victory-portal/victory-portal";
+import Background from "../victory-primitives/background";
```

Comment:
Since we never want to render circular backgrounds here, maybe we can get away with using `Rect` for the background component

Reply:
I will investigate about seeing we can just use `Rect` instead of `Background`